perception/object_merger/include/object_association_merger/node.hpp

```diff
@@ -67,10 +69,12 @@ class ObjectAssociationMergerNode : public rclcpp::Node
   std::unique_ptr<DataAssociation> data_association_;
   std::string base_link_frame_id_;  // associated with the base_link frame
   bool remove_overlapped_unknown_objects_;
+  std::vector<double> distance_threshold_list_;
```

Comment:
no need?

Reply:
This is necessary to use [here](https://github.com/autowarefoundation/autoware.universe/blob/b5f7402abeb3321e4561fe269a975a662bda20de/perception/object_merger/src/object_association_merger/node.cpp#L86)

Reply:
Related this comment : 
https://github.com/autowarefoundation/autoware.universe/pull/1902#discussion_r977877117

>This is necessary to use [here](https://github.com/autowarefoundation/autoware.universe/blob/b5f7402abeb3321e4561fe269a975a662bda20de/perception/object_merger/src/object_association_merger/node.cpp#L86)

Yes. But I think this is no need to implement member variable.

Reply:
Thanks! Sorry for my misunderstanding...
Now I follow your comment, and I'll change it! Thanks a lot!! 👍

Reply:
Remove `distance_threshold_list_` in https://github.com/autowarefoundation/autoware.universe/commit/a057f66fde777e636f65840494a1fda45989df20